### PR TITLE
Pt 162206445 list inventory

### PR DESCRIPTION
**Pull request by valerifilipov**
Get a grouped list of ansible inventory. Seed nodes are marked with `*`

### CODE REVIEW

File: ansible/dump_inventory.py [hunk 1]
```diff
@@ -32,5 +33,6 @@ def natural_sort_key(s, _nsre=re.compile('([0-9]+)')):
         host_ip = host
         if 'ansible_host' in hostvars[host]:
             host_ip = hostvars[host]['ansible_host']
-        print("- %s (%s) [status](http://%s:3013/v2/status) [top](http://%s:3013/v2/blocks/top)" % (host, host_ip, host_ip, host_ip))
+        bullet = '*' if 'kind' in hostvars[host]['tags'] and hostvars[host]['tags']['kind'] == 'seed' else '-'
```
Comment (dincho) [inline ansible/dump_inventory.py]: How this render in markdown ? The output is and should be kept as valid markdown. Once rendered, users should be able to differentiate between the two type of node.

Alternative implementation would be subgroups: seeds, peers, this implementation is good tho, given the above concerns are not valid.

File: Makefile [hunk 2]
```diff
@@ -145,10 +145,19 @@ ifndef DEPLOY_ENV
 	$(error DEPLOY_ENV is undefined)
 endif
 
+dump-inventory:
```
Comment (dincho) [inline Makefile]: Isn't this the equivalent (mostly) to ?:
```
ansible/inventory-list.json:
 	cd ansible && ansible-inventory --list > inventory-list.json
```

Let's try to use the makefile features where possible, I know we're abusing it, but still

File: README.md [hunk 3]
```diff
@@ -66,6 +66,16 @@ Check that your AWS credentials are setup and dynamic inventory is working as ex
...
 
+### List inventory
+
+Get a grouped list of ansible inventory. Seed nodes are marked with `*`
+
+```
+make list-inventory
+```
+
+Inventory data is stored in local file `ansible/inventory-list.json`. To refresh it later you can `make dump-inventory`
```
Comment (dincho) [inline README.md]: I don't think the second sentence is valid, isn't the current implementation only dump it if the file does not exists yet?

File: Makefile (hunk 2 above)
Comment (valerifilipov) [reply Makefile]: Yes, it's the better way, i chose the other way because of shorter name, will change it

File: ansible/dump_inventory.py (hunk 1 above)
Comment (valerifilipov) [reply ansible/dump_inventory.py]: Sorry, never thought output actually should be in some valid format, will change it

File: ansible/dump_inventory.py [hunk 4]
```diff
@@ -24,13 +24,27 @@ def natural_sort_key(s, _nsre=re.compile('([0-9]+)')):
...
     for host in data[group_name]['hosts']:
-        host_ip = host
-        if 'ansible_host' in hostvars[host]:
-            host_ip = hostvars[host]['ansible_host']
-        print("- %s (%s) [status](http://%s:3013/v2/status) [top](http://%s:3013/v2/blocks/top)" % (host, host_ip, host_ip, host_ip))
+        if 'kind' in hostvars[host]['tags'] and hostvars[host]['tags']['kind'] == 'seed':
+            seeds.append(host)
+        else:
+            peers.append(host)
+    print('seeds:')
```
Comment (dincho) [inline ansible/dump_inventory.py]: let's add some bold formatting to this sections:
```suggestion
    print('**seeds:**')
```

File: ansible/dump_inventory.py [hunk 5]
```diff
@@ -24,13 +24,27 @@ def natural_sort_key(s, _nsre=re.compile('([0-9]+)')):
...
+    for seed in seeds:
+        print_host(seed)
+    print('peers:')
```
Comment (dincho) [inline ansible/dump_inventory.py]: same as above
```suggestion
    print('**peers:**')
```

File: ansible/dump_inventory.py (hunk 4 above)
Comment (dincho) [inline ansible/dump_inventory.py]: let's not print this section header if there are no seed peers in that environment

**Comment by dincho** [comment]
> Could of cosmetic changes, otherwise it's working as expected.

s/could/couple

**Pull request merged**